control/veda_deliver_youtube.py

```diff
@@ -1,32 +1,41 @@
-
-import os
-import os.path
-import sys
-import time
-import pysftp
-
 """
 Youtube Dynamic Upload
 
-Note: This represents early VEDA work, but is functional
+Note: This is early VEDA work, but is functional. Ideally deprecated in favor of a no-youtube workflow, this code
+is only maintained, and not prioritized for refactoring
 
 """
+
+import logging
+import os.path
+import pysftp
+import paramiko
+import time
+
 from control_env import *
 
+LOGGER = logging.getLogger(__name__)
+# TODO: Remove this temporary logging to stdout
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
 
 def printTotals(transferred, toBeTransferred):
     """
+    Optional upload logging method
+
+    """
+    '''
     try:
         sys.stdout.write('\r')
         sys.stdout.write("Transferred: {0}\tOut of: {1}\r".format(transferred, toBeTransferred))
         sys.stdout.flush()
     except:
-        print 'Callback Failing'
-    """
-    return None
+        LOGGER.error('Callback Failing')
+    '''
+    return
 
 
-class DeliverYoutube():
+class DeliverYoutube:
```

Comment:
Sorry, missed this place. Could you make it ‘class DeliverYoutube(object):’

Reply:
I went through and cleaned up the others as well -- there's some places where we can probably use inheritance as well, but in the interest of delivering I'll end my refactorfest 😄